src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package frc.team2412.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import static frc.team2412.robot.Hardware.*;
+
+public class IntakeSubsystem extends SubsystemBase{
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_FAST_OUT_SPEED = -0.7;
+
+    CANSparkMax intakeMotorBack;
+    CANSparkMax intakeMotorRight;
+    CANSparkMax intakeMotorFront;
+    CANSparkMax intakeMotorLeft;
+
+    CANSparkMax conveyorBeltOne;
+    CANSparkMax conveyorBeltTwo;
```

Comment:
Nit: should be `private final` (should not be used outside this class and should not be modified).

Reply:
Ight.